exporters/otlp/otlpmetric/otlpmetrichttp/options.go

```diff
@@ -0,0 +1,126 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package otlpmetrichttp
+
+import (
+	"crypto/tls"
+	"time"
+
+	"go.opentelemetry.io/otel/exporters/otlp/otlpmetric/internal/otlpconfig"
+)
+
+const (
+	// defaultMaxAttempts describes how many times the driver
+	// should retry the sending of the payload in case of a
+	// retryable error.
+	defaultMaxAttempts int = 5
+	// defaultMetricsPath is a default URL path for endpoint that
+	// receives metrics.
+	defaultMetricsPath string = "/v1/metrics"
+	// defaultBackoff is a default base backoff time used in the
+	// exponential backoff strategy.
+	defaultBackoff time.Duration = 300 * time.Millisecond
+)
+
+// Compression describes the compression used for payloads sent to the
+// collector.
+type Compression otlpconfig.Compression
+
+const (
+	// NoCompression tells the driver to send payloads without
+	// compression.
+	NoCompression = Compression(otlpconfig.NoCompression)
+	// GzipCompression tells the driver to send payloads after
+	// compressing them with gzip.
+	GzipCompression = Compression(otlpconfig.GzipCompression)
+)
+
+// Option applies an option to the HTTP client.
+type Option interface {
+	applyHTTPOption(*otlpconfig.Config)
+}
+
+// RetrySettings defines configuration for retrying batches in case of export failure
+// using an exponential backoff.
+type RetrySettings otlpconfig.RetrySettings
```

Comment:
This doesn't seems to be in use, although I would like that both metrics and traces HTTP exporters was configured with the `RetrySettings` options, that would make both exporters more uniform with the gRPC exporters.

Reply:
Great thanks for these 2 suggestions, I've already fixed them.